core/src/main/kotlin/io/rover/campaigns/core/events/domain/Event.kt

```diff
@@ -14,4 +14,15 @@ data class Event(
         name: String,
         attributes: Attributes
     ): this(name, attributes, Date(), UUID.randomUUID())
+
+    companion object {
+        fun screenViewedEvent(screenName: String, screenLabel: String? = null, contentID: String? = null): Event {
+            val attributes = mutableMapOf<String, Any>("screenName" to screenName)
+
+            screenLabel?.let { attributes.put("screenLabel", it) }
+            contentID?.let { attributes.put("contentID", contentID) }
```

Comment:
should you use `it` here? you're grabbing the optional value out of the outer scope, by accident maybe?

Reply:
Yeah, thanks should be `it`